api/agent/call.go

```diff
@@ -284,6 +286,11 @@ type call struct {
 
 	// LB & Pure Runner Extra Config
 	extensions map[string]string
+
+	// time.Time equivalents of model Call CreatedAt/StartedAt/CompletedAt
+	createTime   time.Time
+	startTime    time.Time
+	completeTime time.Time
```

Comment:
CreatedAt, et al, are all common.DateTime which is an alias for time.Time, the only difference being the marshal/string methods to use our preferred format, this should not strip monotonic time which would make these unnecessary, no? the main problem currently being we have to set eg CreatedAt on a runner and not on an lb?